### PR TITLE
[typo](docs)add hive catalog faq

### PR DESCRIPTION
**Pull request by hf200012**
add hive catalog faq

# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/lakehouse/multi-catalog/faq.md [hunk 1]
```diff
@@ -96,4 +96,7 @@ under the License.
     ```
 
     可以尝试在 `create catalog` 语句中添加 `"metastore.filter.hook" = "org.apache.hadoop.hive.metastore.DefaultMetaStoreFilterHookImpl"` 解决。
+10. 通过 Hive Catalog 连接 Hive 数据库报错：`RemoteException: SIMPLE authentication is not enabled.  Available:[TOKEN, KERBEROS]`
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/faq.md:99]: 上面空一行

File: docs/zh-CN/docs/lakehouse/multi-catalog/faq.md [hunk 2]
```diff
@@ -96,4 +96,7 @@ under the License.
...
+
+   如果在 `show databases` 和 `show tables` 都是没问题的情况下，查询的时候出现上面的错误，我们需要在 BE 节点执行 Kerberos 的 kinit 然后再重启 BE ，然后再去执行查询即可.
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/faq.md:101]: 增加一项：be/conf 目录下需放置 core-site.xml 和 hdfs-site.xml

Comment (hf200012) [reply docs/zh-CN/docs/lakehouse/multi-catalog/faq.md]: OK

File: docs/zh-CN/docs/lakehouse/multi-catalog/faq.md (hunk 1 above)
Comment (hf200012) [reply docs/zh-CN/docs/lakehouse/multi-catalog/faq.md:100]: OK

**Review by luzhijing — APPROVED** [review]
LGTM

**Comment by luzhijing** [comment]
run buildall

**Pull request merged**